index.html

```diff
@@ -42,7 +43,8 @@ <h4> Avatar is used to represent a user or Profile picture</h4></div>
                             </div>
                         </div>
                         <div class="code">
-                            <iframe src="https://www.thiscodeworks.com/embed/61f93b271e1df40015379416" style="width: 100%; height: 151px;" frameborder="0"></iframe>
+                            <iframe src="https://www.thiscodeworks.com/embed/61f93b271e1df40015379416"
```

Comment:
Use prettier code formatter or 
do force formatting by using ctrl+shift+p (open command pallete) and select force format to make code for readable

Reply:
I used a prettier code formatter now. Thanks

Reply:
I used code formatter, Thanks